pkg/health_probes/defaults.go

```diff
@@ -0,0 +1,32 @@
+package health_probes
+
+import (
+	"fmt"
+
+	n "github.com/Azure/azure-sdk-for-go/services/network/mgmt/2018-12-01/network"
+)
+
+func GetDefaultProbeName(agPrefix string) string {
+	return fmt.Sprintf("%sdefaultprobe", agPrefix)
+}
+
+func GetDefaultProbe(agPrefix string) n.ApplicationGatewayProbe {
```

Comment:
why did we create a package for this ? 😄

Reply:
Avoiding cyclic dependencies and starting to walk down the path of each App Gwy config resource residing in a separate package. Let me prepare a small proposal for this idea.